NSeleneTests/Integration/SharedDriver/SeleneCollection_Enumerable_Should.cs

```diff
@@ -0,0 +1,40 @@
+﻿using NSelene;
+using NSelene.Tests.Integration.SharedDriver.Harness;
+using static NSelene.Selene;
+using NUnit.Framework;
+using System.Linq;
+
+namespace NSeleneTests.Integration.SharedDriver.SeleneCollectionSpec
+{
+    [TestFixture]
+    public class SeleneCollection_Enumerable_Should : BaseTest
+    {
+
+        [TearDown]
+        public void TeardownTest()
+        {
+            Configuration.Timeout = 4;
+        }
+
+        [Test]
+        public void SeleneCollection_Enumerable_ShouldReturnFirst()
```

Comment:
SeleneCollection_Enumerable_Should.SeleneCollection_Enumerable_ShouldReturnFirst

the full name have duplication... let's remove SeleneCollection_Enumerable_Should from the test method name

Reply:
Done